com.unity.ml-agents/Editor/DemonstrationDrawer.cs

```diff
@@ -72,16 +72,17 @@ static string BuildIntArrayLabel(SerializedProperty actionSizeProperty)
         /// </summary>
         void MakeActionsProperty(SerializedProperty property)
         {
-            var actSizeProperty = property.FindPropertyRelative("VectorActionSize");
-            var actSpaceTypeProp = property.FindPropertyRelative("VectorActionSpaceType");
+            var actSpecProperty = property.FindPropertyRelative("VectorActionSpec");
```

Comment:
Should we move all of our strings to constants?  Feels ripe for typo bugs.

Reply:
Moved them all to const strings